docs/en/stack/data-frames/dataframe-limitations.asciidoc

```diff
@@ -0,0 +1,48 @@
+[[ml-df-limitations]]
+== {dataframe-cap} limitations
+
+beta[]
+
+The following limitations and known problems apply to the 7.2 release of 
+the Elastic {dataframe} feature:
+
+[float]
+[[df-limitations-wizard]]
+=== {dataframe-cap} wizard limitation
+
+The {dataframe} wizard in {kib} cannot display the value of a filed selected by 
+using the source index table on the right side if the selected field contains an 
+array.
+
+[float]
+[[df-limitations-ccs]]
+=== {ccs-cap} limitation
+
+{ccs-cap} is not supported in 7.2 for {dataframe-transforms}.
+
+[float]
+[[df-limitations-kibana]]
+=== Displaying {dataframe-transforms} in {kib} limitation
+
+{kib} displays {dataframe-transforms} up to 100.
+
+[float]
+[[df-limitations-dateformat]]
+=== Date histogram limitation
+
+When a date histogram is used as a group by field, the defined interval and the 
+used time format must have the same time fidelity, otherwise it might cause issues 
+in the {dataframe}.
+
+For example, if you set the interval of the date histogram to one minute (1m), 
+then make sure that the time format is `yyyy-MM-dd HH:mm` instead of 
+`yyyy-MM-dd HH:00`.
+
+[float]
+[[df-limitations-aggresponse]]
+=== Strings in aggregation responses
+
+If an aggregation response contains a string (for example, due to a typo in the 
+field name that results in a "NaN" response), the {dataframe} will fail to index 
+the value without warning or causing an error in the {es} logs. However, it results 
+in an empty destination index.
```

Comment:
This sounds like a bug that we are hopefully planning to fix. If so, perhaps we can include a link to the issue where that's tracked?